lua/docs-view.lua

```diff
@@ -1,38 +1,35 @@
 local M = {}
-local config = {
+local default_cfg = {
```

Comment:
if this is named `default_cfg` then there should be another table called just `cfg` which will be the result of the merge between `default_cfg` and `user_cfg`

Reply:
I agree; when I originally wrote these modifications, I attempted to use `tbl_extend`. The reason that I did not follow through with it is because 

1. I think it would require that the proposed `cfg` table is created at the module level instead of inside the setup function because the command itself relies upon a table from the module level.
2. the setup function, as it is now, already updates the `default_cfg` (or, as it is currently named, `config`) and I didn't want to change that because I figured that you wanted it to be that way

If we want to create a new table that is the result of the joining, I don't know how we'd manage to get it to the user command inside of the setup function just yet. I'll have to look into it. I don't know if user commands take default arguments that aren't from the command line.

Reply:
Here is a quick idea of what I am thinking
```lua
local default_cfg = {
  width = 60,
  height = 10,
  position = "left", 
}
local cfg = {}

M.setup = function(user_cfg)
  cfg = vim.tbl_extend("force", default_cfg, user_cfg); // and use `cfg` in the `toggle` function
end
```

Reply:
This makes sense, I don't know why I didn't recognize that I could declare at the module level and assign at the function level. Good idea 👍🏻 I will work on that now

Reply:
Are you comfortable with `default_cfg` being moved to the setup function? It is no longer used outside of it and that would give it the tightest scope.

Reply:
yeah that sounds good